theme/static/style/style.css

```diff
@@ -400,11 +436,15 @@ pre code {
 
 
     /* Screenshots section */
-    .screenshots .img-index {
+    .screenshots .col-2 .img-index {
```

Comment:
There  has to be a comma between `.col-2` and `.img-index {`, so the whole thing will look like `.screenshots .col-2, .img-index {`, as demanded by @atodorov.